CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 0.2.8-rust-1.44.0
```

Comment:
Not sure if this should be a patch version bump or a minor one (I presume since we are under 1.0 even patch update can introduce breaking changes?)